### PR TITLE
added blogs details

### PR DESCRIPTION
**Pull request by Vipul4439**

### CODE REVIEW

**Comment by naresh-io** [comment]
@Vipul4439 Please add the blog somewhere in the middle, not at the bottom or top.

**Comment by Vipul4439** [comment]
Sure

File: data.js [hunk 1]
```diff
@@ -3579,6 +3579,24 @@ module.exports = [
     twitter: "@wuniverse_site",
     tags: ["PHP", "JavaScript", "CSS", "WordPress"],
   },
+
+  {
+    name: "Vipul Soni",
+    description: "Full Stack Web Developer.Write blogs about Web Developement",
+    url: "https://medium.com/@vipulsoniab17_40156",
+    twitter: "https://twitter.com/Vipulso06279659",
```
Comment (naresh-io) [inline data.js:3587]: Pleas fix your twitter handle, it should be `@Vipulso06279659` not the entire URL

Comment (naresh-io) [inline data.js:3587]: Please update your Twitter handle. It should be just your handle `@Vipulso06279659`

**Review by naresh-io — CHANGES_REQUESTED** [review]

**Comment by naresh-io** [comment]
@Vipul4439 can you please resolve the merge conflicts

**Pull request merged**